packages/nextjs-kit/src/__tests__/snapshotTests/__snapshots__/contentWithImage.test.tsx.snap

```diff
@@ -0,0 +1,100 @@
+// Vitest Snapshot v1
+
+exports[`<ContentWithImage /> > should render 'contentWithImage' 1`] = `
+<DocumentFragment>
+  <article
+    class="prose lg:prose-xl mt-10 mx-auto"
+  >
+    <h1>
+      Example Post with Image
+    </h1>
+    <p
+      class="text-sm text-gray-600"
+    >
+      Fri Jun 17 2022
+    </p>
+    <a
+      class="font-normal"
+      href="/"
+    >
+      Back →
+    </a>
+    <div
+      class="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg"
+    >
+      <div
+        class="relative w-full rounded-lg shadow-lg overflow-hidden mb-10"
+        style="height: 50vh;"
+      >
+        <span
+          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
+        >
+          <img
+            alt="Featured Image"
+            data-nimg="fill"
+            decoding="async"
+            sizes="100vw"
+            src="/_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=3840&q=75"
+            srcset="/_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.iohttps%3A%2F%2Fdev-decoupled-wordpress-qa.pantheonsite.io&w=3840&q=75 3840w"
```

Comment:
Kind of surprised this works without mocking `next/image`; looking at the path it doesn't look like it would resolve correctly.